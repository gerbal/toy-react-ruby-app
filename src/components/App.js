import React from "react";
import PropTypes from "prop-types";
import CheckEmail from "./CheckEmail";

const App = ({ name }) => {
  return (
    <div>
      <CheckEmail />
    </div>
  );
};

App.propTypes = {
  name: PropTypes.string
};

export default App;
