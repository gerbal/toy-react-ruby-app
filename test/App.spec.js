import React from 'react';
import { shallow } from 'enzyme';

import App from '../src/components/App';

describe('Component: App', () => {
  it('should render the App component', () => {
    const wrapper = shallow(<App />);

    expect(wrapper).toMatchSnapshot();
  });
});
