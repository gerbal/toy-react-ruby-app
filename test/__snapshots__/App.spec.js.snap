// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: App should render the App component 1`] = `
<div>
  <Connect(CheckEmail) />
</div>
`;
